<ui version="4.0" >
 <class>ExodusIIPanel</class>
 <widget class="QWidget" name="ExodusIIPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="pqSelectionTreeWidget" name="Variables" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Variables</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Range(s)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="ApplyDisplacements" >
       <property name="layoutDirection" >
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text" >
        <string>Apply Displacements</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DisplacementMagnitude" />
     </item>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>mag.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="pqSelectionTreeWidget" name="Blocks" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Blocks</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="pqSelectionTreeWidget" name="Sets" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Sets</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="pqSelectionTreeWidget" name="SetResults" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Set Results</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="pqCollapsedGroup" name="modeShapesGroup" >
     <property name="title" >
      <string>Mode Shapes</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="HasModeShapes" >
        <property name="text" >
         <string>Has Mode Shapes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ModeShapeOptions" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>2</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>Mode:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>181</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="ModeLabel" />
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
             <item>
              <widget class="QSlider" name="ModeSelectSlider" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="ModeSelectSpinBox" />
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="pqSelectionTreeWidget" name="Maps" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Maps</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="pqCollapsedGroup" name="FilePatternGroup" >
     <property name="title" >
      <string>File Pattern</string>
     </property>
     <property name="collapsed" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>File Prefix</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="FilePrefix" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="FileRange_1" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="FileRange_0" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>First file index</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Last file index</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="FilePattern" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>File Pattern</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QCheckBox" name="ExodusModelMetadata" >
        <property name="text" >
         <string>Meta Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqSelectionTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqSelectionTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>pqCollapsedGroup</class>
   <extends>QGroupBox</extends>
   <header>pqCollapsedGroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
